# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/pkg_patch/gcc-index.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- gcc-4.1.1/libstdc++-v3/include/ext/rope.orig	2006-12-14 22:24:49.000000000 -0300
+++ gcc-4.1.1/libstdc++-v3/include/ext/rope	2006-12-14 22:25:32.000000000 -0300
@@ -66,6 +66,8 @@
 
 #include <ext/memory> // For uninitialized_copy_n
 
+#undef index
+
 namespace __gnu_cxx
 {
   using std::size_t;
--- gcc-4.1.1/libstdc++-v3/include/ext/ropeimpl.h.orig	2006-12-14 22:24:54.000000000 -0300
+++ gcc-4.1.1/libstdc++-v3/include/ext/ropeimpl.h	2006-12-14 22:26:08.000000000 -0300
@@ -53,6 +53,8 @@
 #include <ext/memory> // For uninitialized_copy_n
 #include <ext/numeric> // For power
 
+#undef index
+
 namespace __gnu_cxx
 {
   using std::size_t;
